Bootstrap.php

```diff
@@ -103,6 +104,10 @@ public function bootstrap($app)
             }
 
             Yii::$container->set('dektrium\user\Mailer', $module->mailer);
+
+            // Ensure the module is not in DEBUG mode on production environments
+            if(!defined('YII_DEBUG') || (defined(YII_DEBUG) && YII_DEBUG == false))
```

Comment:
Code needs to follow PSR-2 codestyle. So we need space after `if` and brackets around if body.